Libraries/Components/Touchable/Touchable.js

```diff
@@ -312,6 +314,15 @@ var TouchableMixin = {
     this.pressOutDelayTimeout && clearTimeout(this.pressOutDelayTimeout);
   },
 
+  propTypes: {
```

Comment:
Do propTypes work on Mixins? Also can you make sure the website picks this up and show the proper doc. If not you might have to copy the new propType in all the Touchable*

You can run it locally
```
cd website
npm install
npm start
```